.gitignore

```diff
@@ -3,10 +3,13 @@
 *.png
 *.xyz
 *.pdb
+*.ipynb
 
 # Signac
 .signac_sp_cache.json.gz
 signac_project_document.json
+signac_statepoints.json
```

Comment:
Is it supposed to be `statepoints` with an “s”?

Reply:
It is on my end! When you run init.py, do you get a differently named file?

Reply:
It did not appear to do that for me. Maybe there is a specific command to export all statepoints to a single json? Either way, that is fine.